testpar/t_cache.c

```diff
@@ -7103,5 +7105,5 @@ main(int argc, char **argv)
     MPI_Finalize();
 
     /* cannot just return (failures) because exit code is limited to 1byte */
-    return (failures != 0);
+    return (nerrors != 0 || failures != 0);
```

Comment:
This is a little weird, but setup errors and test errors are tracked separately.